macrosynergy/panel/category_relations.py

```diff
@@ -533,6 +549,62 @@ def ols_table(self):
 
 if __name__ == "__main__":
 
+    # Notebook investigation
+
+    cids_g3 = ["EUR", "JPY", "USD"]
+    cids_dmsc_du = ["AUD", "CAD", "CHF", "GBP", "NOK", "NZD", "SEK"]
+    cids_dmsc_eq = ["AUD", "CAD", "CHF", "GBP", "SEK"]
+    cids_dmdu = cids_g3 + cids_dmsc_du
+    cids_dmeq = cids_g3 + cids_dmsc_eq
+    ms = "CPIC_SJA_P6M6ML6AR_XR" # "CPIH_SA_P1M1ML12_XR"  # main signal
+
+    dict_eqdi = {
+        "sig": ms,
+        "rivs": None,
+        "targ": "EQXR_VT10",
+        "cidx": ["EUR", "USD"],
+        "black": None,
+        "srr": None,
+        "pnls": None,
+    }
+
+    path_to_feather = "C:/Users/RSueppel/OneDrive/Documents/Business/Macrosynergy/" \
+                      "notebooks/classified/data/feathers/"
+    dfx = pd.read_csv(f"{path_to_feather}dfx_infov.csv")
```

Comment:
Needs to be changed to a generic path before merge to develop

Reply:
done thanks